src/lib/Components/Bidding/Screens/SelectMaxBid.tsx

```diff
@@ -1,45 +1,27 @@
 import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
 import styled from "styled-components/native"
 
 import { Button } from "../Components/Button"
 import { MaxBidPicker } from "../Components/MaxBidPicker"
 import { Title } from "../Components/Title"
 
-interface SelectMaxBidProps {
+import { SelectMaxBid_saleArtwork } from "__generated__/SelectMaxBid_SaleArtwork.graphql"
+
+interface SelectMaxBidProps /* extends ViewProperties */ {
   saleArtworkID: string
+  sale_artwork: SelectMaxBid_saleArtwork
 }
 
-// we are using hard-coded values for now.
-const Bids = [
-  {
-    label: "$35,000 USD",
-    value: 3500000,
-  },
-  {
-    label: "$40,000 USD",
-    value: 4000000,
-  },
-  {
-    label: "$45,000 USD",
-    value: 4500000,
-  },
-  {
-    label: "$50,000 USD",
-    value: 5000000,
-  },
-  {
-    label: "$55,000 USD",
-    value: 5500000,
-  },
-]
-
 export class SelectMaxBid extends React.Component<SelectMaxBidProps> {
   render() {
+    // TODO metaphysics should return formatted values
+    const bids = this.props.sale_artwork.bid_increments.map(d => ({ label: d.toString(), value: d }))
     return (
       <Container>
         <Title>Your max bid</Title>
 
-        <MaxBidPicker selectedValue={4500000} bids={Bids} />
+        <MaxBidPicker selectedValue={4500000} bids={bids} />
```

Comment:
If you want to add slightly to the scope of this PR, the initial `selectedValue` should be the first increment in the list, and subsequent selections should be based on what the user picks

Reply:
I added `selectedValue` just to make the initial screen look like the mockup (sorry for the noisy and confusing interface). I'd be happy to drop it and change `<MaxBidPicker>` to default to the first increment.

Reply:
Makes sense!

Reply:
Actually it defaults to the first value when `selectedValue` is not passed. So I guess I will just remove that.